application/managers/applications.py

```diff
@@ -269,18 +274,38 @@ async def get_component_health(self, component: Component, application: Applicat
         """
         Returns component health condition.
         """
-        try:
-            component_health_status = await self.helm_manager.release_health_status(
-                application.organization,
-                application.context_name,
-                application.namespace,
-                component.name
-            )
-            health_status = component_health_status['status']
-            details = component_health_status['details']
-        except ReleaseNotFoundException:
-            health_status = ReleaseHealthStatuses.unhealthy
-            details = {}
+        match component.type:
```

Comment:
Nice one! I like it!
Consider to add `case _:` to get rid of unexpected behavior.